### PR TITLE
remove var log

### PR DESCRIPTION
**Pull request by ericgaspar**
- No need to creat `/var/log/$app/$app.log` path, it should be handled by `ynh_use_logrotate` helper.

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/install [hunk 1]
```diff
@@ -176,6 +163,14 @@ pushd "$final_path"
 	ynh_exec_warn_less ynh_exec_as $app bin/mattermost team add "$team_name" "$admin"
 popd
 
+#=================================================
+# SETUP LOGROTATE
+#=================================================
+ynh_script_progression --message="Configuring log rotation..." --weight=1
+
+# Use logrotate to manage application logfile(s)
+ynh_use_logrotate
```
Comment (kemenaran) [inline scripts/install:172]: Will it also do the `chown $app -R $logs_path` part?

**Review by kemenaran — APPROVED** [review]

File: scripts/install (hunk 1 above)
Comment (tituspijean) [reply scripts/install:172]: Looks like not: https://github.com/YunoHost/yunohost/blob/b452838a1797641f82778e60f4e50e1722837eba/data/helpers.d/logrotate#L97

**Comment by ericgaspar** [comment]
!testme

**Comment by ericgaspar** [comment]
Closing. Logrotate helper as changed and invalidates this PR

**Pull request closed without merging**